src/v2/view-builder/views/shared/ChallengeOktaVerifyCustomAppDataView.js

```diff
@@ -1,17 +1,16 @@
+// Common view for OV push and custom push.
```

Comment:
custom push -> custom app

And is this view for only OV Push? Or can it be for other methods of OV? If so, lets put the comment as `Common view for OV and Custom App`